src/usdb_syncer/notes_parser.py

```diff
@@ -409,3 +425,24 @@ def minimum_song_length(self) -> str:
         )
 
         return f"{minutes:02.0f}:{seconds:02.0f}"
+
+
+def _fix_polish_letters(string: str) -> str:
+    return (
+        string.replace("¹", "ą")
+        .replace("¥", "Ą")
+        .replace("æ", "ć")
+        .replace("Æ", "Ć")
+        .replace("ê", "ę")
+        .replace("Ê", "Ę")
+        .replace("³", "ł")
+        .replace("£", "Ł")
+        .replace("ñ", "ń")
+        .replace("Ñ", "Ń")
+        .replace("œ", "ś")
+        .replace("Œ", "Ś")
+        .replace("Ÿ", "ź")
+        .replace("Ź", "Ź")
+        .replace("¿", "ż")
+        .replace("¯", "Ż")
```

Comment:
Maybe this could go into a constant we iterate over:
```py
POLISH_REPLACEMENTS = (("¥", "Ą"), ("æ", "ć"), ...)
```
and then
```py
for old, new in POLISH_REPLACEMENTS:
    string = string.replace(old, new)
```
This should be easier to extend if we add more language fixes later.